packages/atomic/src/components/facets/atomic-facet/atomic-facet.tsx

```diff
@@ -76,24 +77,35 @@ export class AtomicFacet
    */
   @Prop() public label = 'No label';
   /**
-   * The character that separates values of a multi-value field
+   * The character that separates values of a multi-value field.
    */
   @Prop() public delimitingCharacter?: string;
   /**
    * The number of values to request for this facet. Also determines the number of additional values to request each time this facet is expanded, and the number of values to display when this facet is collapsed.
    */
   @Prop() public numberOfValues = 10;
+  /**
+   * Whether this facet should contain a search box.
+   */
+  @Prop() public enableFacetSearch = false;
```

Comment:
should be true by default here, category facet is less subject/designed for a search for it's ok that its false

Reply:
Oh, I though the JSUI category facet had `enableFacetSearch` to `false` by default because it was added later and we did not want this to be breaking?

I think the facet search feature should be opt-out rather than opt-in, unless UX really does not want the facet search to be shown with the category?

Reply:
It's a bit of this too, but I remember Karen saying that this facet was not designed to have a search as it's more for exploration by drilling down into the category than search for an exact value.